src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/scenari/PushPlatformValidationScenario.scala

```diff
@@ -0,0 +1,49 @@
+package org.apache.james.gatling.jmap.rfc8621.scenari
+
+import io.gatling.core.Predef.{exec, _}
+import io.gatling.core.structure.ScenarioBuilder
+import io.gatling.http.Predef._
+import org.apache.james.gatling.control.RecipientFeeder.RecipientFeederBuilder
+import org.apache.james.gatling.control.UserFeeder.UserFeederBuilder
+import org.apache.james.gatling.jmap.draft.CommonSteps.provisionSystemMailboxes
+import org.apache.james.gatling.jmap.rfc8621.JmapEmail.{nonEmptyListMessagesChecks, openpaasEmailQueryParameters, queryEmails}
+import org.apache.james.gatling.jmap.rfc8621.JmapHttp.{noError, statusOk}
+import org.apache.james.gatling.jmap.rfc8621.JmapWebsocket.{enablePush, websocketClose, websocketConnect}
+import org.apache.james.gatling.jmap.rfc8621.scenari.EmailChangesScenario.emailIds
+import org.apache.james.gatling.jmap.rfc8621.scenari.PushPlatformValidationScenario.{emailState, mailboxState}
+import org.apache.james.gatling.jmap.rfc8621.{JmapEmail, JmapMailbox, SessionStep}
```

Comment:
There's a trap here:

We do use RFC-8621 in INBOX for:
 - Listing mailboxes (`Mailbox/get`)
 - Doing /changes calls
 
 But we do use draft for...
  - listing emails
  - getting emails

The scenario should then mix both protocols to reflect reality.